qualtran/_infra/controlled_test.py

```diff
@@ -124,7 +126,7 @@ def _test_cirq_equivalence(bloq: Bloq, gate: cirq.Gate):
     bloq = Controlled(Swap(5), CtrlSpec(qdtypes=QUInt(4), cvs=0b0101))
     quregs = get_named_qubits(bloq.signature)
     ctrl, x, y = quregs['ctrl'], quregs['x'], quregs['y']
-    circuit1, _ = bloq.decompose_bloq().to_cirq_circuit(None, **quregs)
+    circuit1, _ = bloq.decompose_bloq().to_cirq_circuit_and_quregs(None, **quregs)
```

Comment:
do you wanna go through these (where the second return value is ignored) and use the new method?

Reply:
done